docs/plugins/redis.md

```diff
@@ -0,0 +1,30 @@
+# redis
+
+Connects to a local (by default) redis instance and stores a `redis`
+connection handle at `server.notes.redis`. 
+
+## Config
+
+The `redis.ini` file has two sections (defaults shown):
+
+### [server]
+
+    host=127.0.0.1
+    ; port=6379
+    ; db=0
+
+### [redisOpts]
+
+    ; see https://www.npmjs.com/package/redis#overloading
+
+
+## Usage
+
+Use redis in your plugin like so:
+
+    if (connection.server.notes.redis) {
```

Comment:
For consistency; everywhere else we use `server.notes` rather than `connection.server.notes` as server is exported to the plugin sandbox explicitly.